api/mesh/v1alpha1/virtual_outbound.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package kuma.mesh.v1alpha1;
+
+option go_package = "github.com/kumahq/kuma/api/mesh/v1alpha1";
+
+import "mesh/v1alpha1/selector.proto";
+import "config.proto";
+
+option (doc.config) = {
+  type : Policy,
+  name : "VirtualOutbound",
+  file_name : "virtual-outbound"
+};
+
+// VirtualOutbound defines a way to generate hostname and ports from existing
+// services.
+message VirtualOutbound {
+
+  // List of selectors to match dataplanes.
+  repeated Selector selectors = 1;
+
+  // Configuration defines settings of the tracing.
+  message Conf {
+    // A gotemplate to generate hostnames for the matching DP.
+    // The template will use the keys in the parameters map.
+    string host = 1;
+    // A gotemplate to generate a port for the matching DP.
+    // The template will use the keys in the parameters map.
+    string port = 2;
+    // A map from name to tag names.
+    // Keys will be parameters for the gotemplate of host or port.
+    map<string, string> parameters = 3;
```

Comment:
`parameters` sounds a bit unobvious in my opinion, it doesn't highlight the fact this is a mapping from parameter's name to tag's name until you read the docs. In the proposal, it was `tags` even though it is a more complicated abstraction than just `tags`, I like `tags`. 

If we want to go with `parameters` maybe it makes sense to highlight the mapping to tag's name:
```yaml
parameters:
  - name: port
    tagKey: port
  - name: service
    tagKey: kuma.io/service
```
I don't insist, it is just an idea, what do you think?

Reply:
I'd rather stay away from tags are they may confuse people.
I like your idea of a map I'll try this out